apps/ledger-live-mobile/src/screens/Onboarding/steps/useCaseSelection.tsx

```diff
@@ -169,6 +178,32 @@ const OnboardingStepUseCaseSelection = () => {
     ],
   );
 
+  const onCloseProtectDrawer = useCallback(() => {
+    setIsProtectDrawerOpen(false);
+  }, []);
+
+  const onBuyNanoX = useCallback(() => {
+    Linking.openURL("https://shop.ledger.com/pages/ledger-nano-x");
+
+    track("button_clicked", {
```

Comment:
Better set the tracking event directly on the Button component
See this https://github.com/LedgerHQ/ledger-live/wiki/LLM:Analytics#track-events

And normally you don't have to set the source or timestamp properties in the track event, it's automatic
See this : https://github.com/LedgerHQ/ledger-live/wiki/LLM:Analytics